### DISCUSSION TITLE
Starter Keypoints of Interest

### DISCUSSION THREAD

**Discussion by JRoGueDZYN** [akatori-game/feedback#116] [Feedback]
To start off with not exactly sure if this is the correct point of where I should be posting this information but here goes:
When starting the game the first thing I get greeted with is "An error occurred while updating Akatori Demo (missing executable): C:\Program Files (X86)\Steam\steamapps\common\Akatori demo\deathstick_devm1.exe

When this took place I went into said folder and the only executable file was the akatori.exe which I assume is what the actual file that supposed to be run. Instead of worrying about running through the Steam window I just started the game via the akatori.exe file itself and began the game. 

I'm not exactly sure what the specs are that this game needs but for some reason right out the bat I'm greeted with a 20 fps, and even lowering the settings that doesn't change. Is there something that's going on via my end or something on the software side?

The game so far is rather enjoyable even if it is a bit slow going due to 20 fps but it's playable. Let me know if there are any changes I can make on my end to correct any issues if said issues are mine. Thank you for this opportunity and I look forward to further playing on my end. Keep up the great work!

**Comment by XenoWarrior** [comment]
Hey @JRoGueDZYN 

Thanks for raising this up, luckily we're aware of it!

Bug ticket(s):
Steam: https://github.com/akatori-game/feedback/issues/117
Performance: https://github.com/akatori-game/feedback/issues/114
